src/shared/components/toggle-button/index.module.css

```diff
@@ -0,0 +1,46 @@
+@import "shared/styles/imports/variables.css";
+
+.switch {
+    position: relative;
+    width: 60px;
+    height: 34px;
+    display: inline-block;
+
+    & .slider {
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        background-color: #ccc;
```

Comment:
I know these are not the final styles, just don't forget to put this into a variable later 😇